topic-offset-service/src/main/java/uk/gov/gchq/palisade/service/topicoffset/model/TopicOffsetRequest.java

```diff
@@ -33,13 +33,13 @@
 import java.util.StringJoiner;
 
 /**
- * The TopicOffsetRequest is the input for topic-offset-service.  This is used to optimise the later retrieval of the
+ * The TopicOffsetRequest is the input for Topic-Offset Service. This is used to optimise the later retrieval of the
  * data for the client. TopicOffsetResponse is the output for this service.
  * Note there are three classes that effectively represent the same data but under a slightly different transformation:
  * <ul>
- * <li> {@code uk.gov.gchq.palisade.service.attributemask.message.AttributeMaskingResponse} is the output from the attribute-masking-service.
- * <li> {@code uk.gov.gchq.palisade.service.topicoffset.message.TopicOffsetRequest} is the input for this the topic-offset-service.
- * <li> {@code uk.gov.gchq.palisade.service.filteredresource.message.FilteredResourceRequest} is the input for this the filtered-resource-service.
+ * <li> uk.gov.gchq.palisade.service.attributemask.model.AttributeMaskingResponse is the output from the Attribute-Masking Service.
+ * <li> {@link TopicOffsetRequest} is the input for this the Topic-Offset Service.
+ * <li> uk.gov.gchq.palisade.service.filteredresource.model.FilteredResourceRequest is the input for this the Filtered-Resource Service.
```

Comment:
Keep the `@code` formatting.
The last point is irrelevant in this context.
```suggestion
 * <li> {@code uk.gov.gchq.palisade.service.attributemask.model.AttributeMaskingResponse} is the output from the Attribute-Masking Service.
 * <li> {@link TopicOffsetRequest} is the input for this the Topic-Offset Service.
```